src/main/java/legend/game/combat/types/AdditionCharEffectData0c.java

```diff
@@ -0,0 +1,35 @@
+package legend.game.combat.types;
+
+import legend.core.memory.Value;
+import legend.core.memory.types.MemoryRef;
+import legend.core.memory.types.ShortRef;
+import legend.core.memory.types.UnsignedByteRef;
+
+public class AdditionCharEffectData0c implements MemoryRef {
+  private final Value ref;
+
+  public final UnsignedByteRef scrolling_00; /**1 when character is scrolling, 0 when set**/
+
+  public final ShortRef dupes_02; /**Number of dupe characters to render to create motion blur effect**/
+  public final ShortRef position_04; /**Character print timing, stored as a negative int**/
+  public final ShortRef offsetY_06; /**Offset from base y-coordinate**/
+  public final ShortRef offsetX_08; /**Offset from base x-coordinate**/
+  public final ShortRef offsetY_0a; /**Offset from base y-coordinate (seems to be dupe)**/
```

Comment:
Javadoc goes before the thing it's attached to like
```suggestion
  /** Offset from base y-coordinate (seems to be dupe) **/
  public final ShortRef offsetY_0a;
```

Reply:
Hmm, some of these comments seem like they're for things that would be caught by a linter. :wink: